### DISCUSSION TITLE
Add wikipedia article

### DISCUSSION THREAD

**Discussion by mullimanko** [Stirling-Tools/Stirling-PDF#1489] [General]
There is a wikipedia article "List of PDF software" (https://en.wikipedia.org/wiki/List_of_PDF_software) and I tried to add Stirling-PDF there as a bullet point under the heading "Web-based". However it is only possible to add software that has its own wikipedia article. So, my wish is to create a wikipedia article for Stirling-PDF and then add Stirling-PDF to https://en.wikipedia.org/wiki/List_of_PDF_software. This would increase its visibility.

**Comment by Frooodle** [comment]
@mullimanko  Good idea! Please do, you can reach out to me on discord if you need any information etc and I'm happy to do any writeups to help on this

**Comment by Frooodle** [comment]
Also not sure if we are web based, multi platform or both 😂

**Comment by Frooodle** [comment]
> There is a wikipedia article "List of PDF software" (https://en.wikipedia.org/wiki/List_of_PDF_software) and I tried to add Stirling-PDF there as a bullet point under the heading "Web-based". However it is only possible to add software that has its own wikipedia article. So, my wish is to create a wikipedia article for Stirling-PDF and then add Stirling-PDF to https://en.wikipedia.org/wiki/List_of_PDF_software. This would increase its visibility.

So err.. how's that going? 😂

**Comment by mullimanko** [comment]
I tried to publish the wikipedia article but the article was declined by an editor with the reason that it needs multiple references that are in-depth (not just passing mentions about the subject), reliable, secondary, and independent of the subject. I only know one suitable reference and used that. Do you have more references that can be used for the article? In the following is the draft:  
  
_Stirling-PDF is a [free and open-source](https://en.wikipedia.org/wiki/Free_and_open-source_software) PDF editor that can handle typical PDF tasks (e.g., merge, rotate, rearrange, and remove pages) and is operated via a self-hosted web interface[[1]](https://en.wikipedia.org/w/index.php?title=User:Trunkensoul/sandbox&action=edit#cite_note-:0-1). Stirling-PDF does not contact external services, which means confidential data can also be processed[[1]](https://en.wikipedia.org/w/index.php?title=User:Trunkensoul/sandbox&action=edit#cite_note-:0-1). The software started as being created to a large extent with the help of Chat-GPT, which was at the beginning of 2023._

_References_
_¹Mahn, Jan (2023-12-01). ["PDF-Universalist"](https://www.heise.de/select/ct/2023/28/2330716361459296684). c't (in German). Vol. 2023, no. 28. p. 90. [ISSN](https://en.wikipedia.org/wiki/ISSN_(identifier)) [0724-8679](https://search.worldcat.org/issn/0724-8679). Retrieved 2024-07-20._

**Comment by Frooodle** [comment]
Our github as well as 
https://docs.stirlingpdf.com/
and
stirlingpdf.com
should contain various info 

There are also some interviews such as
https://blog.pixee.ai/user-spotlight-stirling-pdf

or reviews/blogs out there too

**Reply by mullimanko** [reply]
The github site and the website are not secondary and independent of the subject. And the blog https://blog.pixee.ai/user-spotlight-stirling-pdf doesn't work due to the following paragraph in the wikipedia article [Reliable sources](https://en.wikipedia.org/wiki/Wikipedia:Reliable_sources#User-generated_content):

> Content from websites whose content is largely [user-generated](https://en.wikipedia.org/wiki/User-generated_content) is generally unacceptable. Sites with user-generated content include personal websites, personal and [group](https://en.wikipedia.org/wiki/Group_blog) blogs (excluding [newspaper and magazine blogs](https://en.wikipedia.org/wiki/Wikipedia:Verifiability#Newspaper_and_magazine_blogs)), [content farms](https://en.wikipedia.org/wiki/Content_farm), [Internet forums](https://en.wikipedia.org/wiki/Internet_forum), [social media sites](https://en.wikipedia.org/wiki/Social_networking_service), [fansites](https://en.wikipedia.org/wiki/Fansite), [video](https://en.wikipedia.org/wiki/Online_video_platform) and [image](https://en.wikipedia.org/wiki/Image_hosting_service) hosting services, most [wikis](https://en.wikipedia.org/wiki/Wiki) and other collaboratively created websites.

Feel free to write reliable sources here if they comply with the wikipedia article [Reliable sources](https://en.wikipedia.org/wiki/Wikipedia:Reliable_sources#User-generated_content).
